### PR TITLE
DEV-3502-feat(piscine-go): add README.md for new go exercise podiumposition

### PR DESCRIPTION
**Pull request by tiagocollot**
Created and added README.md for new go exercise podiumposition.

### CODE REVIEW

File: subjects/podiumposition/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+## podiumposition
+
+### Instructions
+
+A F1 race just finished and the commentator is calling the finishing positions incorrectly.
+Help to fix this before the contestants arrive at the podium by providing the commentator with the correct podium position.
```
Comment (jotapero) [inline subjects/podiumposition/README.md:6]: Sugestion: "You need to help fix this before .... "

File: subjects/podiumposition/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Write a function `PodiumPosition` that accepts an array of arrays as a `string` and returns the competitor positions correctly.
```
Comment (jotapero) [inline subjects/podiumposition/README.md:8]: Write a function `PodiumPosition` that takes an array of arrays of type `string` and returns ...

**Review by jotapero — CHANGES_REQUESTED** [review]

File: subjects/podiumposition/README.md (hunk 1 above)
Comment (tiagocollot) [reply subjects/podiumposition/README.md:6]: It is good suggestion, but I think it also works well the way it is now.

**Review by jotapero — APPROVED** [review]

**Review by jotapero — APPROVED** [review]

File: subjects/podiumposition/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+A F1 race just finished and the commentator is calling the finishing positions incorrectly.
+Help to fix this before the contestants arrive at the podium by providing the commentator with the correct podium position.
+
+Write a function `PodiumPosition` that takes an array of arrays of type `string` and returns the competitor positions correctly.
+
+### Expected function
+
+```go
+func PodiumPosition(podium[][] string) [][]string {
```
Comment (eslopfer) [inline subjects/podiumposition/README.md:13]: The space for the parameter is in the wrong place

File: subjects/podiumposition/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+## podiumposition
+
+### Instructions
+
+A F1 race just finished and the commentator is calling the finishing positions incorrectly.
+Help to fix this before the contestants arrive at the podium by providing the commentator with the correct podium position.
+
+Write a function `PodiumPosition` that takes an array of arrays of type `string` and returns the competitor positions correctly.
```
Comment (eslopfer) [inline subjects/podiumposition/README.md:8]: Does it take an array of arrays or a slice of slices? arrays or slices? https://go.dev/tour/moretypes/6

**Review by eslopfer — CHANGES_REQUESTED** [review]

File: subjects/podiumposition/README.md (hunk 3 above)
Comment (tiagocollot) [reply subjects/podiumposition/README.md:13]: It is fixed.

File: subjects/podiumposition/README.md (hunk 4 above)
Comment (tiagocollot) [reply subjects/podiumposition/README.md:8]: Thank you Esteban for noticing it, it is a slice of slices and it's fixed!

**Review by jotapero — APPROVED** [review]
All good, nice catch

**Review by eslopfer — APPROVED** [review]
LGTM! :rocket:

**Pull request merged**